pkg/operator/controller/garden/garden/reconciler.go

```diff
@@ -172,9 +175,11 @@ func (r *Reconciler) reportProgress(log logr.Logger, garden *operatorv1alpha1.Ga
 
 func (r *Reconciler) updateStatusOperationStart(ctx context.Context, garden *operatorv1alpha1.Garden, operationType gardencorev1beta1.LastOperationType) error {
 	var (
-		now                           = metav1.NewTime(r.Clock.Now().UTC())
-		description                   string
-		mustRemoveOperationAnnotation bool
+		now                = metav1.NewTime(r.Clock.Now().UTC())
+		operations         = helper.GetGardenerOperations(garden.Annotations)
+		operationsSet      = sets.New(operations...)
+		filteredOperations = slices.Clone(operationsSet.UnsortedList())
```

Comment:
I think `Clone` here is unnecessary

```suggestion
		filteredOperations = operationsSet.UnsortedList()
```